components/Buttons/examples/ButtonsShapesExampleViewController.m

```diff
@@ -41,55 +42,50 @@ - (void)viewDidLoad {
   self.view.backgroundColor = [UIColor colorWithWhite:0.9f alpha:1];
   UIColor *titleColor = [UIColor whiteColor];
 
+  MDCButtonScheme *buttonScheme = [[MDCButtonScheme alloc] init];
+  buttonScheme.colorScheme = self.colorScheme;
+  buttonScheme.typographyScheme = self.typographyScheme;
+
   // Raised button
 
-  MDCRaisedButton *raisedButton = [[MDCRaisedButton alloc] init];
-  [raisedButton setTitle:@"Add To Cart" forState:UIControlStateNormal];
-  [MDCButtonTypographyThemer applyTypographyScheme:self.typographyScheme toButton:raisedButton];
+  MDCButton *containedButton = [[MDCButton alloc] init];
+  [containedButton setTitle:@"Add To Cart" forState:UIControlStateNormal];
+  [MDCContainedButtonThemer applyScheme:buttonScheme toButton:containedButton];
 
   UIImage *plusImage = [UIImage imageNamed:@"Plus"];
   plusImage = [plusImage imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
-  [raisedButton setImage:plusImage forState:UIControlStateNormal];
-  UIColor *titleAndImageColor =
-      [UIColor colorWithRed:71/255.0f green:50/255.0f blue:50/255.0f alpha:1];
-  [raisedButton setTitleColor:titleAndImageColor forState:UIControlStateNormal];
-  [raisedButton setImageTintColor:titleAndImageColor forState:UIControlStateNormal];
-  [raisedButton setBackgroundColor:
-      [UIColor colorWithRed:242/255.0f green:174/255.0f blue:178/255.0f alpha:1]];
+  [containedButton setImage:plusImage forState:UIControlStateNormal];
 
   MDCRectangleShapeGenerator *raisedShapeGenerator =
       [[MDCRectangleShapeGenerator alloc] init];
   [raisedShapeGenerator setCorners:[[MDCCutCornerTreatment alloc] initWithCut:8.f]];
-  raisedButton.shapeGenerator = raisedShapeGenerator;
+  containedButton.shapeGenerator = raisedShapeGenerator;
 
-  [raisedButton sizeToFit];
-  [raisedButton addTarget:self
+  [containedButton sizeToFit];
+  [containedButton addTarget:self
                    action:@selector(didTap:)
          forControlEvents:UIControlEventTouchUpInside];
-  [self.view addSubview:raisedButton];
+  [self.view addSubview:containedButton];
 
   // Disabled raised button
 
-  MDCRaisedButton *disabledRaisedButton = [[MDCRaisedButton alloc] init];
```

Comment:
unfortunately these removals caused the example in dragons to have the buttons colorless. I had these buttons have specific colors to showcase the vignette buttons. We should probably fix this at least so that in dragons they will have colors